modules/policy-criteria.adoc

```diff
@@ -274,11 +274,19 @@ AND, OR
 6+| *Section: Container configuration*
 
 | Environment Variable
-| Check environment variables by name or value.
+| Check environment variables by name or value. When you create a policy that includes the environment variable attribute,
+you can choose which types of environment variables the policy should match. For example, you can specify raw values, which are provided directly in the deployment YAML, or you can specify references to values from ConfigMaps, Secrets, fields, or resource requests or limits.
+For any type other than a raw value specified directly in the deployment YAML, the corresponding `value` attribute of the policy rule is ignored. In this case, the policy match is evaluated on the existence of the specified environment variable type. Additionally, this criteria disallows the creation of policies with a non-empty `value` attribute for types other than raw values.
 | Environment Variable
-| RAW=key=value to match an environment variable as directly specified in the deployment configuration with a specific key and value. `value` can be omitted to match on only the key. +
+a| RAW=key=value to match an environment variable as directly specified in the deployment configuration with a specific key and value. `value` can be omitted to match on only the key. +
```

Comment:
I'd use deployment configuration or deployment YAML consistently. Pick one - I really can't say which one is better but if I were to guess - deployment configuration? (deployment YAML speaks to an engineer though :-))

Reply:
Our style guide is against "deployment configuration" so I think YAML is better.